tools/compile-translations.go

```diff
@@ -80,17 +80,19 @@
 
 package main
 
-import "encoding/binary"
-import "bufio"
-import "bytes"
-import "fmt"
-import "io/ioutil"
-import "log"
-import "os"
-import "os/exec"
-import "path/filepath"
-import "sort"
-import "strings"
+import (
```

Comment:
Undo unrelated changes like this one.

Reply:
golangci-lint makes the multiple import to a group of imports

Reply:
even the `staticcheck` does it which is the default formatter for golang